scripts/build_emscripten.sh

```diff
@@ -34,7 +34,7 @@ else
     BUILD_DIR="$1"
 fi
 
-# solbuildpackpusher/solidity-buildpack-deps:emscripten-13
+# solbuildpackpusher/solidity-buildpack-deps:emscripten-14
```

Comment:
There's a bug here. The hash below has not been updated. Looks like [the nightly build in solc-bin has been failing since Friday](https://github.com/ethereum/solc-bin/actions/runs/3560588983/jobs/5980711895) due to this:
```
-- Found Z3: /emsdk/upstream/emscripten/cache/sysroot/usr/lib/cmake/z3/Z3Config.cmake (found version "4.11.0.0") 
CMake Error at CMakeLists.txt:84 (message):
  SMTChecker tests require Z3 4.11.2 for all tests to pass.

  Build with -DSTRICT_Z3_VERSION=OFF if you want to use a different version.
  You can also use -DUSE_Z3=OFF to build without Z3.  In both cases use
  --no-smt when running tests.


-- Configuring incomplete, errors occurred!
```

Pinging @r0qs.

Reply:
True! Made a PR https://github.com/ethereum/solidity/pull/13757

Reply:
Thanks, forgot about it.